js/models/message_handler.js

```diff
@@ -0,0 +1,72 @@
+import { convertOptionNamesToCamelCase } from "../utils";
+import A from "../imports";
+
+export default class MessageHandler {
+  constructor(aladin) {
+    this.aladin = aladin;
+  }
+
+  handleChangeFoV(msg) {
+    this.aladin.setFoV(msg["fov"]);
+  }
+
+  handleGotoRaDec(msg) {
+    this.aladin.gotoRaDec(msg["ra"], msg["dec"]);
+  }
+
+  handleAddCatalogFromURL(msg) {
+    const options = convertOptionNamesToCamelCase(msg["options"] || {});
+    this.aladin.addCatalog(A.catalogFromURL(msg["votable_URL"], options));
+  }
+
+  handleAddMOCFromURL(msg) {
+    const options = convertOptionNamesToCamelCase(msg["options"] || {});
+    if (options["lineWidth"] === undefined) options["lineWidth"] = 3;
```

Comment:
I do not think it is needed anymore. In aladin lite there is a default linewidth given if you do not explicitely give one. The default is indeed 3